src/Compilers/CSharp/Portable/Syntax/StackAllocArrayCreationExpressionSyntax.cs

```diff
@@ -15,8 +15,18 @@ public partial class LocalDeclarationStatementSyntax
         public LocalDeclarationStatementSyntax Update(SyntaxTokenList modifiers, VariableDeclarationSyntax declaration, SyntaxToken semicolonToken)
             => Update(default(SyntaxToken), modifiers, declaration, semicolonToken);
     }
+
+    public partial class LocalDeclarationStatementSyntax
```

Comment:
This is identical to the above declaration. Won't compile.

Reply:
Merge conflict caused it, fixing